person.rb

```diff
@@ -1,11 +1,13 @@
 require './nameable'
+require './rental'
 class Person < Nameable
```

Comment:
- [ ] The `attr_accessor :name, :age, :rentals` should be present inside the class person.Kindly add it.
- [ ] There should be a function to create rental in person.rb file.Kindly add it 
```
 def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```